validphys2/src/validphys/coredata.py

```diff
@@ -317,10 +318,16 @@ def multiplicative_errors(self):
         mult_systype = self.systype_table[self.systype_table["type"] == "MULT"]
         # NOTE: Index with list here so that return is always a DataFrame, even
         # if N_sys = 1 (else a Series could be returned)
-        mult_table = self.systematics_table.loc[:, ["MULT"]]
-        # Minus 1 because iloc starts from 0, while the systype counting starts
-        # from 1.
-        mult_table = mult_table.iloc[:, mult_systype.index - 1]
+        mult_table = self.systematics_table.filter(like = "MULT")
+        # TODO: check whether the `filter` creates a problem with n_sys = 1, if not remove both
+        # note and todo
+        #mult_table = self.systematics_table.loc[:, ["MULT"]]
+
+        if self.legacy:
+            # Minus 1 because iloc starts from 0, while the systype counting starts
+            # from 1.
+            mult_table = mult_table.iloc[:, mult_systype.index - 1]
```

Comment:
Why is this done for legacy datasets only?

Reply:
The uncertainties in the new format are either ADD or MULT. In the previous format you had all the uncertainties in one file and then another file which was the one deciding what they were (and that's why the two steps were needed)